modules/oe_translation_poetry/src/Plugin/tmgmt/Translator/PoetryTranslator.php

```diff
@@ -247,13 +247,16 @@ public function getRoutes(): RouteCollection {
    * @SuppressWarnings(PHPMD.CyclomaticComplexity)
    */
   public function jobItemFormAlter(array &$form, FormStateInterface $form_state): void {
-    // Improve the button labels.
+
+    // Improve the button labels and custom redirect page.
     if (isset($form['actions']['accept'])) {
       $form['actions']['accept']['#value'] = t('Accept translation');
+      $form['actions']['accept']['#submit'][] = [__CLASS__, 'jobItemFormRedirect'];
     }
 
     if (isset($form['actions']['save'])) {
       $form['actions']['save']['#value'] = t('Update translation');
+      $form['actions']['save']['#submit'][] = [__CLASS__, 'jobItemFormRedirect'];
```

Comment:
Not very usual form to referring to the class, in Drupal the preferred method seems to be either ['::jjobItemFormRedirect'] or [$this, 'jobItemFormRedirect']. 
This change is not a hard requirement, just food for thought.